ecom-config.yaml

```diff
@@ -28,25 +28,28 @@ githubRepo: https://github.com/vippsas/vipps-ecom-api
 resources:
   - title: Documentation
     slug: documentation
-    description: Read the documentation of our API
+    description: Read the documentation of our ecom API
     markdownUrl: vipps-ecom-api/master/vipps-ecom-api.md
     githubUrl: https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md
   - title: FAQ
     slug: faq
     description: Check the frequently asked questions
     markdownUrl:  vipps-ecom-api/master/vipps-ecom-api-faq.md
     githubUrl: https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md
-  - title: Getting Started
-    slug: getting-started
-    description: Start here, if you plan to integrate with our API
-    markdownUrl: vipps-developers/master/vipps-getting-started.md
+  - title: Checklist
+    slug: checklist
+    description: Checklist for going live in production
+    markdownUrl: vipps-ecom-api/blob/master/vipps-ecom-api-checklist.md
```

Comment:
Please remove the `blob` from the URL, as it is mapped to `https://raw.githubusercontent.com/vippsas/`.
The raw url does not contain the blob part, and it would result in a 404.